poker/.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
the same https://github.com/GoodFellow23/SilentTeam-PP-FE/pull/4/files#r702038984